modules/ROOT/pages/platform/security/encryption.adoc

```diff
@@ -63,8 +48,14 @@ For more information see the xref:auradb/importing/import-database.adoc#_neo4j_a
 To clone an instance protected by a Customer Managed Key, the key must be valid and available to Aura.
 The cloned instance, by default, uses the available Customer Managed Key for that region and product.
 
-You can override this behavior by selecting the Neo4j Managed Key when cloning the database.
-If there is no valid CMK for the destination region and product, the Neo4j Managed Key is used to encrypt the cloned instance.
+It is best practice to use the same CMK key as the instance it’s being cloned from. 
+You can override this to use another CMK key - but you can not use the Neo4j Managed Key.
+
+=== Detach a CMK from Aura
```

Comment:
I find "remove" clearer than "detach" but thats just my opinion and I dont feel strongly. So feel free to leave as is.